### ISSUE TITLE
Support for EPUB3 SMIL feature

### ISSUE THREAD

**Issue by arky** [learningequality/kolibri#4376]
We plan to create listen and learn digital books for pre-school children. These books will use media overlay SMIL spec in EPUB3. Please kindly include a web interface capable of displaying such EPBU3 books in Kolibri. 

Currently [Readium project](https://readium.org/) provides JavaScript tools and web interface to make this possible. 

Here is example of such 'learn and learn' digital book in action: 

https://www.readbeyond.it/readium/?epub=epub_content/c6aa1093

**Comment by indirectlylit** [comment]
Thanks @arky. For the upcoming 0.11 we used epub.js, which does not seem to support SMIL out of the box.

@christianmemije was Readium one of the libraries we considered, and if so do you recall our motivations for choosing not to use it?

**Comment by rtibbles** [comment]
@indirectlylit yes, we considered Readium, the documentation was not great, and the API not very well defined, it is not out of the question to transition, but it would probably take a bit of work to do so.

**Comment by arky** [comment]
Thanks @indirectlylit @rtibbles . Perhaps future versions of epub.js will support SMIL?

**Comment by rtibbles** [comment]
Not sure - but we'll keep an eye out!

**Comment by arky** [comment]
I have filed a upstream epubjs-reader bug requesting support for SMIL support. 
https://github.com/futurepress/epubjs-reader/issues/21
